src/slider/slider.component.ts

```diff
@@ -129,13 +122,30 @@ export class Slider implements AfterViewInit, OnDestroy, ControlValueAccessor {
 	/** Used to generate unique IDs */
 	private static count = 0;
 	/** The lower bound of our range */
-	@Input() min = 0;
+	@Input() set min(v) {
+		this._min = v;
+		this.value = this.value;
```

Comment:
Does this do anything?

Reply:
Yeah 😅 it forces the component to update ... I'll add a comment 👍